src/Authentication/appsettings.json

```diff
@@ -29,5 +29,14 @@
     "CertificatePath": "jwtselfsignedcert.pfx"
   },
   "OidcProviders": {
+  },
+  "QueueStorageSettings": {
```

Comment:
Have the QueueStorageSettings been added to keyvault for deploy to the different environments?

Reply:
No, I'm yet to create the values.yaml PR